<ui version="4.0" >
 <class>WOptimizedExecutionOrderUI</class>
 <widget class="QDialog" name="WOptimizedExecutionOrderUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Optimal Execution Order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTableWidget" name="execution_table_p" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="rowCount" >
      <number>0</number>
     </property>
     <column>
      <property name="text" >
       <string>Execution Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Coverage</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="progress_label_p" >
       <property name="text" >
        <string>Progress:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_p" >
       <property name="value" >
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_p" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
